test/yayson/presenter.coffee

```diff
@@ -282,6 +282,16 @@ describe 'Presenter', ->
     expect(json.data.relationships.car.links.related).to.eq '/cars/3/car'
     expect(json.data.relationships.car.data).to.eq undefined
 
+  it 'should not include relationships with nothing in them', ->
+    class CarPresenter extends Presenter
+      type: 'cars'
+
+      relationships: ->
+        car: CarPresenter
+
+    json = CarPresenter.render(id: 3)
+    expect(json.data.relationships.car).to.eq undefined
```

Comment:
Given the spec http://jsonapi.org/format/#document-resource-object-relationships maybe this should instead of `undefined` be {data: null} since you try to represent a car that isnt linked to another car? Right now it looks like it doesn't have a relationship to `car`, but actually it does. It is just a relationship to no car :)